packages/ember-flight-icons/package.json

```diff
@@ -34,50 +34,52 @@
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
+    "@babel/core": "^7.22.20",
     "@hashicorp/flight-icons": "^2.20.0",
     "ember-auto-import": "^2.6.3",
-    "ember-cli-babel": "^7.26.11",
-    "ember-cli-htmlbars": "^6.2.0"
+    "ember-cli-babel": "^8.0.0",
+    "ember-cli-htmlbars": "^6.3.0"
   },
   "devDependencies": {
+    "@babel/eslint-parser": "^7.22.15",
     "@ember/optional-features": "^2.0.0",
-    "@ember/string": "^3.1.1",
-    "@ember/test-helpers": "^2.9.3",
-    "@embroider/test-setup": "^2.1.1",
+    "@ember/test-helpers": "^3.2.0",
+    "@embroider/test-setup": "^3.0.1",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
-    "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-cli": "~4.12.1",
-    "ember-cli-dependency-checker": "^3.3.1",
+    "ember-cli": "~5.3.0",
+    "ember-cli-clean-css": "^3.0.0",
+    "ember-cli-dependency-checker": "^3.3.2",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
     "ember-disable-prototype-extensions": "^1.1.3",
+    "ember-get-config": "^2.1.1",
```

Comment:
if this is needed by the addon code should it go in `dependencies`?

Reply:
yes, it probably should – was playing to see it if fixes the issue, and it seems like it does so will move it to dependencies

Reply:
(sorry for the early review, more of a TIL that this was even a thing for me)

Reply:
feedback is welcome anytime! I fixed this one, but will push with the other changes to give the CI some breathing space)